R/time_bp_to_index.R

```diff
@@ -10,6 +10,18 @@
 time_bp_to_index <- function(time_bp, time_steps) {
   # give warning if some dates are too extreme
   time_steps_ordered <- sort(time_steps)
+  # if you have only one time step within your database you cannot calculate the range between min and max time steps
```

Comment:
I think it would be simpler to have
```
if (length(time_steps_ordered) == 1) {
# do range checks
}
 # get indeces
  time_indeces <- 
```
etc. Could you please change accordingly?

Reply:
Done